packages/react-server/core/ClientController.js

```diff
@@ -573,22 +580,18 @@ class ClientController extends EventEmitter {
 
 			CURRENT_STATE_FRAME = frame;
 
-			if (this.framebackController.isActive()){
-				this.framebackController.navigateBack();
-			} else {
-				var frameback = (state||{}).frameback;
-				if (context) {
-					var path = this._history.getPath();
-
-					// Pass in "popstate" because this is
-					// when a user clicks the forward/back
-					// button.
-					context.navigate(
-						new ClientRequest(path, {frameback}),
-						History.events.POPSTATE
-					);
+			var frameback = (state||{}).frameback;
+			if (context) {
+				var path = this._history.getPath();
+
+				// Pass in "popstate" because this is
+				// when a user clicks the forward/back
+				// button.
+				context.navigate(
+					new ClientRequest(path, {frameback}),
+					History.events.POPSTATE
+				);
```

Comment:
This diff chunk looks bigger than it is because of the indentation change.  It's just replacing the `if/else` with what was previously the body of the `else` branch (eliminating the `if` case).